src/foam/u2/wizard/common/ReviewWizardletView.js

```diff
@@ -19,12 +33,28 @@ foam.CLASS({
 
   methods: [
     function render() {
-      this.addClass()
-        .start('h2').add('review view').end()
-        .forEach(this.items, function (item) {
-          // TODO
-        })
-        ;
+      var self = this;
+      this.SUPER();
+      this.addClass(this.myClass())
+        .start()
+          .start().addClass('h200').add(this.TITLE).end()
+          .start().addClass('generic-container')
+            .forEach(this.items, function (item) {
+              if ( item.noData )  {
+                this.start(item.border).tag(item.view).end();
+                return;
+              }
+                // TODO: uncomment this line once we have data
+//              else if ( ! self.data.value[item.name] ) return;
```

Comment:
these should be uncommented before we merge
```suggestion
              else if ( ! self.data.value[item.name] ) return;
```